docs/content/preview/tutorials/azure/azure-app-service.md

```diff
@@ -0,0 +1,140 @@
+1. Execute the `data.sql` script against your cluster.
+
+Now that your cluster is seeded with data, it's time to deploy and run the application on Azure.
+
+## Deploy YB Shoes to Azure App Service
+
+Azure provides multiple deployment methods for your applications.
+
+Since we'll be deploying our application from a local git repository, let's use the Azure CLI. It's also worth noting that Azure has a powerful extension for Visual Studio Code, which can execute the same commands.
+
+First, we need to [configure our application settings](https://learn.microsoft.com/en-us/cli/azure/webapp/config/appsettings?view=azure-cli-latest) in Azure.
+
+```sh
+# Convert the downloaded CA certificate from YugabyteDB Managed to a single line string, then Base64 encode it
+# Azure Configuration Settings forbid special characters, so this ensures the cert can be passed properly to our application
+# Tip: Run this command to convert cert file to base64 encoded single line string:
+# cat /path/to/cert/file | base64
+
+az webapp config appsettings set -g GROUP_NAME -n APPLICATION_NAME --setting DB_HOST=[YB_DB_HOST] DB_USERNAME=admin DB_PASSWORD=[YB_DB_PASSWORD] DB_CERTIFICATE=[BASE_64_ENCODED_DB_CERTIFICATE]
+```
+
+After you execute this command, you can view the configuration settings in the Azure web portal.
+
+![App configuration settings in Azure web portal.](/images/tutorials/azure/azure-app-service/azure-configuration.png "App configuration settings in Azure web portal.")
+
+Next, use the Azure CLI to deploy your application code to the cloud, as follows:
+
+1. Install the Azure CLI.
+1. In the Azure Deployment Center, configure your Local Git / FTPS Credentials under User Scope
+1. Configure the project for local git deployment as follows:
+
+    ```sh
+    az webapp deployment source config-local-git --name APPLICATION_NAME --resource-group RESOURCE_GROUP
+    ```
+
+    This outputs the git URL, similar to the following:
+
+    ```output
+    {
+    "url": "https://bhoyer@yb-shoe-sales.scm.azurewebsites.net/yb-shoe-sales.git"
+    }
+    ```
+
+1. Set git remote for Azure using the URL from previous output.
+
+    ```sh
+    git remote add azure https://bhoyer@yb-shoe-sales.scm.azurewebsites.net/yb-shoe-sales.git
+    ```
+
+1. Push to the Azure remote.
+
+    ```sh
+    git push azure main:master
+    ```
+
+After pushing code to the remote repository, Azure automatically builds and publishes the application. Once this is complete, visit your application's default domain to view YB Shoes, running in the Azure cloud infrastructure!
+
+## Wrapping up
+
+As you see, moving an application to the cloud with Azure App Service and YugabyteDB Managed doesn't require much additional configuration. For further insight into building Node.js applications with YugabyteDB, check out the following blog:
+
+- [Node.js Smart Drivers for YugabyteDB: Why You Should Care](https://www.yugabyte.com/blog/node-js-smart-drivers-for-yugabytedb/)
```

Comment:
This app has been using the smart driver? There are a bunch of caveats for using smart drivers in apps in YBM